### DISCUSSION TITLE
2022-03-02 Weekly Meeting 45 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#190] [Weekly Meetings]
Weekly Meeting 45 is happening on Wednesday March 2nd, 2022 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Announcements and Updates
TBD!
Discuss OMI space to meet and play! https://github.com/omigroup/website/issues/51

Working Group Updates:
-  OMI Website update
   - Interoperator group update?
- OMI-Chan group
- WASM group
- gLTF group
- any others?

Housekeeping:
Any?

Open Floor!
 
Please leave any additional agenda items below in the comments. Thanks!

**Comment by mrmetaverse** [comment]
- Feedback is the life-blood of iteration. I will be hosting another OMI retrospective session. The goal is to share what has been working, what could work better, and create actionable recommendations for moving forward. If successful, we can consider doing this regularly  :calendar: [add to your calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=MzNpcHJhYWJoOWt1cHFxN3YybjY2aGFqanYgY18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bn&tmsrc=c_0dpugf9f83eq4qekmb6omqbjmg%40group.calendar.google.com)
- If we have enough extra time, we can run another Lean Coffee style discussion.

**Reply by mrmetaverse** [reply]
note: event moved back 30 min to 22:30 UTC (5:30 pm ET)

**Comment by robertlong** [comment]
Discuss OMI space to meet and play!
- Connected spaces across platforms
- Already have an OpenSim space
- Can list these spaces on the website
- Need to discuss what platforms to reach out to and include

**Reply by mrmetaverse** [reply]
happy to put a OMI portal/space [in the Cryptovoxels property we have at Angell](https://www.cryptovoxels.com/play?coords=N@1369W,1079S). Also happy to reformat [this CV spaces space](https://www.cryptovoxels.com/spaces/a3b3a554-d25f-4ab0-8c7c-854bc23e4c7f) (which are free) and portal to it from our CV property on Scarcity Island.

**Reply by mrmetaverse** [reply]
btw this ticket was moved to the website group last week, so referencing it takes the format of `omigroup/website#51` rather than `#51` 
ex. omigroup/website#51 ✔️ 
ex. #51 ❌

**Reply by aiaustin** [reply]
Reminder that website list of OMI Spaces should have a line saying who to contact to add an entry.

Also, seeing the OMI Twitter Community now created, maybe that should be added to the website.

**Comment by robertlong** [comment]
Interoperator group
- https://github.com/omigroup/omigroup/issues/188
- Schedule next meeting for March 11th
